yapapi/executor/_smartq.py

```diff
@@ -52,9 +61,17 @@ def data(self) -> Item:
 
 
 class SmartQueue(Generic[Item], object):
-    def __init__(self, items: Iterable[Item], *, retry_cnt: int = 2):
+    def __init__(self, items: Iterable[Item]):
+        """
+        :param items: the items to be iterated over
+        :param retry_cnt:
```

Comment:
If we drop this argument from `__init__()` (I understand that's because it's unused) then let's not add a docstring for it.